app/livechat/server/lib/routing/GuestPool.js

```diff
@@ -0,0 +1,89 @@
+// import { settings } from '../../../settings';
+import { Meteor } from 'meteor/meteor';
+import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
+
+import { Livechat } from '../Livechat';
+import { RoutingManager } from '../RoutingManager';
+import { LivechatInquiry } from '../../../lib/LivechatInquiry';
+import { sendNotification } from '../../../../lib';
+import { Rooms } from '../../../../models';
+
+/* Guest Pool Queuing Method:
+	*
+	* An incomming livechat is created as an Inquiry
+	* which is picked up from an agent.
+	* An Inquiry is visible to all agents (TODO: in the correct department)
+	*
+	* A room is still created with the initial message, but it is occupied by
+	* only the client until paired with an agent
+*/
+class GuestPool {
+	constructor() {
+		this.config = {
+			previewRoom: true,
+			showConnecting: true,
+			showQueue: true,
```

Comment:
according to [this previous code](https://github.com/RocketChat/Rocket.Chat/pull/15003/files#diff-35ad4c8f424bcf2b344f149d439212d4L91) the correct value should be `false`